### PR TITLE
refactor!: remove economy from ecalc

### PR DESCRIPTION
**Pull request by frodehk**
BREAKING CHANGE: Not possible to use TAX and QUOTA for emissions, and PRICE for fuel.

## Why is this pull request needed?

Remove economical calculations from eCalc: reduce technical debt, main focus is energy and emissions - not simplified economical evaluations.

## What does this pull request change?

- [x] Remove `TAX` and `QUOTA` from emissions
- [x] Remove `PRICE` from fuel
- [x] Remove `NORWEGIAN_KRONER` and `NORWEGIAN_KRONER_PER_DAY` from units
- [x] Update yamls
- [x] Update tests
- [x] Update snapshots
- [x] Update schema
- [x] Update docstrings
- [x] Update docs

## Issues related to this change:
https://equinor-ecalc.atlassian.net/browse/ECALC-412?atlOrigin=eyJpIjoiNTlhNDExNjdjZDk5NDg2NTg2YmIxYmQ5YjQ3ZjYxMzgiLCJwIjoiaiJ9

### CODE REVIEW

File: src/libecalc/core/consumers/direct_emitter.py [hunk 1]
```diff
@@ -35,13 +35,10 @@ def __init__(self, direct_emitter_dto: dto.DirectEmitter):
...
                     # Commented out, meaning the fuel model has CATEGORY set to None
                     # user_defined_category=direct_emitter_dto.user_defined_category,
-                    price=None,  # See docstring
                     emissions=[
                         Emission(
                             name=direct_emitter_dto.emission_name,
-                            quota=emitter_model.emission_quota,
-                            factor=Expression.setup_from_expression(1),  # See docstring.
-                            tax=None,  # See docstring
+                            factor=Expression.setup_from_expression(1),  # See docstring.  # See docstring
```
Comment (TeeeJay) [inline src/libecalc/core/consumers/direct_emitter.py:41]: Thats a lot of docstring to look at ...

Comment (TeeeJay) [reply src/libecalc/core/consumers/direct_emitter.py:41]: But, update docstring that refers to cost and tax

**Review by TeeeJay — COMMENTED** [review]
Missing some JSON Schema/YAML changes, along with examples etc, but the changes done looks goo.
Waiting for snapshots. I guess @jsolaas has the complete overview if this is all.. perhaps some JSON

**Comment by frodehk** [comment]
> Missing some JSON Schema/YAML changes, along with examples etc, but the changes done looks goo.
> 
> Waiting for snapshots. I guess @jsolaas has the complete overview if this is all.. perhaps some JSON 

Test- and snapshot updates are on the road. Schema/yaml as well.

File: docs/docs/about/references/keywords/REGULARITY.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ COMPRESSOR|&#9744;|     Stream day      |
 
 ### Reporting
 
-- All fuel rates, tax and emission results are reported in calendar days.
+- All fuel rates are reported in calendar days.
 - All power and volume rates results are reported in stream day rates. Note that the volume rates are only present in the .json file.
 
 The reason for reporting calendar day rate is due to the economic interpretation, or rather the lack of economic
```
Comment (TeeeJay) [inline docs/docs/about/references/keywords/REGULARITY.md:58]: Should this be worded differently? Not referring to economic?

**Review by TeeeJay — APPROVED** [review]
nice, seems correct to me :)

**Review by jsolaas — COMMENTED** [review]
Very nice! Is this breaking in the sense that models with price, tax or quota will not run? If so I think we should be able to 'silently' ignore those keys. Instead we could use this opportunity to implement warnings to user in web.

File: src/libecalc/dto/types.py [hunk 3]
```diff
@@ -112,22 +110,13 @@ class InterpolationType(str, Enum):
 
 class FuelType(EcalcBaseModel):
     """An installation/model/component may change fuel over time, due to setup changes,
-    production, and the cost may also change on different predictive models.
+    production.
```
Comment (olelod) [inline src/libecalc/dto/types.py:113]: Not your fault, I guess, but this sentence in the doctoring does not really make sense to me...

Comment (frodehk) [reply src/libecalc/dto/types.py:113]: I agree, should we just remove it - or do you have a suggestion to improve the doctoring? (e.g. "data transfer object for FuelType"?)

**Comment by frodehk** [comment]
> Very nice! Is this breaking in the sense that models with price, tax or quota will not run? If so I think we should be able to 'silently' ignore those keys. Instead we could use this opportunity to implement warnings to user in web.

It is actually not breaking in that sense. Models with price, tax or quota in the yaml will run. The dto-layer (FuelMapper and EmissionMapper) just returns the allowed parameters, ignoring price, tax or quota. It does not give a warning per now.

**Comment by olelod** [comment]
Maybe remove the units NORWEGIAN_KRONER and NORWEGIAN_KRONER_PER_DAY as well

File: src/libecalc/dto/types.py (hunk 3 above)
Comment (olelod) [reply src/libecalc/dto/types.py:113]: I don't have any good suggestions. Just remove it, I guess.

**Pull request merged**